pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -92,13 +92,11 @@ func evalPorts(exposedPorts []string, node *parser.Node, from, to int) []string
 	envs := evalVars(node, from, to, exposedPorts, shlex)
 	ports := make([]string, 0, len(exposedPorts))
 	for _, p := range exposedPorts {
+		if pp, err := shlex.ProcessWord(p, envs); err == nil {
+			p = pp
+		}
```

Comment:
shouldn't this take some action if there is an error?

Reply:
if `ProcessWord` returns an error, this leaves the original port string unchanged for further processing and if it doesn't get evaluated, `describe.go` will warn about it to the user.

Do we want to log the error here as well? And if so, I assume not the loud warning as in `describe.go` but rather import a logger and do something more subtle

Reply:
this is fine